src/schema/v2/conversation/send_message_mutation.ts

```diff
@@ -1,46 +1,103 @@
-import { GraphQLString, GraphQLNonNull } from "graphql"
+import {
+  GraphQLString,
+  GraphQLNonNull,
+  GraphQLList,
+  GraphQLBoolean,
+} from "graphql"
 import {
   mutationWithClientMutationId,
   cursorForObjectInConnection,
 } from "graphql-relay"
 import { ConversationType, MessageEdge } from "./index"
 import { ResolverContext } from "types/graphql"
+import { GraphQLInputObjectType } from "graphql"
+
+interface SendConversationMessageMutationProps {
+  attachments?: {
+    name: string
+    type: string
+    url: string
+    id?: string
+    size?: string
+  }
+  bodyHTML?: string
+  bodyText: string
+  from: string
+  fromId?: string
+  id: string
+  replyAll?: boolean
+  replyToMessageID: string
+  to?: string[]
+}
 
-export default mutationWithClientMutationId<any, any, ResolverContext>({
+export default mutationWithClientMutationId<
+  SendConversationMessageMutationProps,
+  any,
+  ResolverContext
+>({
   name: "SendConversationMessageMutation",
   description: "Appending a message to a conversation thread",
   inputFields: {
-    id: {
+    attachments: {
+      description: "Attachments to the message",
+      type: new GraphQLList(
+        new GraphQLInputObjectType({
+          name: "ConversationMessageAttachmentInput",
+          fields: {
```

Comment:
I assume this schema makes sense as something coming from Impulse (like `id` isn't required _seems_ maybe off but if it matches Impulse)

Reply:
All that's strictly needed are the the fields required here, though there might be some updating here in near-term.